pkg/minikube/bootstrapper/exec_runner.go

```diff
@@ -48,11 +48,14 @@ func (*ExecRunner) Run(cmd string) error {
 func (*ExecRunner) CombinedOutput(cmd string) (string, error) {
 	glog.Infoln("Run with output:", cmd)
 	c := exec.Command("/bin/bash", "-c", cmd)
-	out, err := c.CombinedOutput()
+	c.Stdout = os.Stdout
+	c.Stderr = os.Stdout
+	err := c.Run()
 	if err != nil {
-		return "", errors.Wrapf(err, "running command: %s\n output: %s", cmd, out)
+		return "", errors.Wrapf(err, "running command: %s\n.", cmd)
 	}
-	return string(out), nil
+
+	return "", nil
```

Comment:
This function is meant to return the combined output. That's the string parameter in the output.

I think we need to do something like you had before, where we capture both stdout and stderr as variables, then run the command and return the combined strings.

Reply:
For a long running command, like continues printing new logs, it is not possible to get the return strings. 

I would like to propose the change like this:

```
func (*ExecRunner) CombinedOutputTo(cmd string, out io.Writer) error {
        glog.Infoln("Run with output:", cmd)
        c := exec.Command("/bin/bash", "-c", cmd)
        c.Stdout = out
        c.Stderr = out
        err := c.Run()
        if err != nil {
                return errors.Wrapf(err, "running command: %s\n.", cmd)
        }
        return nil
}
```

```
func (*ExecRunner) CombinedOutput(cmd string) (string, error) {
        var b bytes.Buffer
        err := CombinedOutputTo(cmd,  &b)
        if err != nil {
		return "", errors.Wrapf(err, "running command: %s\n output: %s", cmd, b.Bytes())
        return b.Bytes(), nil
}
```

For command like `minikube logs -f`, it should call `CombinedOutputTo` directly:
```
        CombinedOutputTo(cmd, os.Stdout)
```